index.d.ts

```diff
@@ -0,0 +1,51 @@
+/**
+Escape a string for use in HTML.
+
+Escapes the following characters in the given `string` string: `&` `<` `>` `"` `'`.
+
+@example
+```
+import * as escapeGoat from 'escape-goat';
+
+escapeGoat.escape('🦄 & 🐐');
+//=> '🦄 &amp; 🐐'
+
+escapeGoat.escape('Hello <em>World</em>');
+//=> 'Hello &lt;em&gt;World&lt;/em&gt;'
+```
+*/
+export function escape(string: string): string;
+
+/**
+Unescape an HTML string to use as a plain string.
+
+Unescapes the following HTML entities in the given `htmlString` string: `&amp;` `&lt;` `&gt;` `&quot;` `&#39;`.
+
+@example
+```
+import * as escapeGoat from 'escape-goat';
```

Comment:
Here, it makes sense to import as named exports as the method names themselves are clear enough without the context of the import.
```suggestion
import {unescape} from 'escape-goat';
```